Makefile

```diff
@@ -25,6 +26,11 @@ setup: pipenv
 setup-dev: pipenv
 	$(PIPENV) install --verbose --dev --python $(PYTHON_VERSION)
 
+setup-ui-test:
```

Comment:
please use `[dev-packages]` for that purpose https://pipenv.pypa.io/en/latest/pipfile.html
the team is using pipenv, let's be consistent and extend:
https://github.com/h2oai/h2o-llmstudio/blob/main/Pipfile#L56

Reply:
@psinger do you have an idea why @itsmunishbhardwaj might have the problem above?

Reply:
not really, but we mainly support Linux and not Mac

Reply:
FYI, mac is not supported